lib/commands/connect/shared.js

```diff
@@ -109,77 +110,69 @@ let request = exports.request = function(token, method, path, data){
   return promise;
 };
 
-let withUserConnections = exports.withUserConnections = function(token, appName, flags, allowNone, heroku){
+let withUserConnections = exports.withUserConnections = co.wrap(function* (token, appName, flags, allowNone, heroku){
```

Comment:
What's interesting is that the caller to `withUserConnection` expects a promise, which it gets either way, because it's already explicitly returning a promise with `co()` (I think?). So I agree that `co.wrap` feels safer, but that's just because it seems like magic sprinkles to me, and you can never have too many magic sprinkles. :sparkles: :smile: :sparkles: